web/core/components/exporter/export-modal.tsx

```diff
@@ -1,6 +1,7 @@
 "use client";
 
 import React, { useState } from "react";
+import _ from "lodash";
```

Comment:
You should import only the methods we need. import intersection from lodash/intersection.